demosplan/DemosPlanCoreBundle/Logic/Statement/StatementDeleter.php

```diff
@@ -192,50 +247,14 @@ public function deleteStatementObject(
                     $success = false;
                 }
             } else {
-                if ($lockedByAssignmentOfRelatedFragments) {
-                    $this->getLogger()->warning("Statement {$statementId} was not deleted, because of related fragments are locked by assignment");
-                    $this->messageBag->add(
-                        'warning',
-                        'warning.delete.statement.because.of.fragments.not.claimed.by.current.user',
-                        ['externId' => $statement->getExternId()]
-                    );
-                }
-
-                if ($lockedByAssignment) {
-                    $this->getLogger()
-                        ->warning("Statement {$statementId} was not deleted, because of locked by assignment");
-                    $this->messageBag->add(
-                        'warning', 'warning.delete.statement.because.of.assignment',
-                        ['externId' => $statement->getExternId()]
-                    );
-                }
-
-                if ($lockedByCluster) {
-                    $this->getLogger()
-                        ->warning("Statement {$statementId} was not deleted, because of locked by cluster");
-                    $this->messageBag->add(
-                        'warning', 'error.statement.clustered.in',
-                        ['headStatementId' => $statement->getExternId()]
-                    );
-                }
-
-                if ($lockedBecauseOfOriginal) {
-                    $this->getLogger()
-                        ->warning("Statement {$statementId} was not deleted, because it is a undeletable original-Statement");
-                    $this->messageBag->add(
-                        'warning', 'warning.delete.statement.original',
-                        ['externId' => $statement->getExternId()]
-                    );
-                }
-
-                if ($lockedBySync) {
-                    $this->getLogger()
-                        ->warning("Statement {$statementId} was not deleted, because of locked by related synced statement.");
-                    $this->messageBag->add(
-                        'warning', 'warning.delete.statement.synced', // add transkey
-                        ['externId' => $statement->getExternId()]
-                    );
-                }
+                $this->getDeleteStatementLogger(
```

Comment:
why did you call it get...? It does not get anything imho

Reply:
done in [bc03a5e](https://github.com/demos-europe/demosplan-core/pull/3014/commits/bc03a5e8e61e1204615996ee644fc38722032288)